browser_history/generic.py

```diff
@@ -254,3 +256,33 @@ def to_csv(self):
             for row in self.get():
                 writer.writerow(row)
             return output.getvalue()
+
+    def to_json(self, json_lines=True):
+        """
+         Return history formatted as a JSON Lines or Plain JSON format
+         names
+         :return: :py:class:`str` object
+        """
+        # custom json encoder for datetime objects
+        class DateTimeEncoder(json.JSONEncoder):
+            # Override the default method
+            def default(self, obj):
+                if isinstance(obj, (datetime.date, datetime.datetime)):
+                    return obj.isoformat()
+
+        # fetch lines
+        lines = []
+        for entry in self.entries:
+            json_record = json.dumps(
+                {self.fields[0]: entry[0], self.fields[1]: entry[1]}, cls=DateTimeEncoder
```

Comment:
You could use a loop/comprehension here instead of hardcoding the number of columns.

Reply:
sure will look into this